### ISSUE TITLE
training

### ISSUE THREAD

**Issue by dujuan1999** [venturezhao/XJTU-Tripler#1]
Hello! Is the training code for the model not provided?
Thanks！

**Comment by venturezhao** [comment]
Hi! Since the training data is too huge, and we just exploited the common training method to cover this job, it is no need to put our training data here.
For the NN training, you may just follow the official ShuffleNet V2 x1 work.

**Comment by francis2tm** [comment]
Hey @dujuan1999 ! Have u managed to replicate this project?
